automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala

```diff
@@ -899,8 +902,9 @@ class RawlsApiSpec extends TestKit(ActorSystem("MySpec")) with AnyFreeSpecLike w
               status should not be("Running")
               metadata
             }
-
-            parseWorkflowStatusFromMetadata(notRunningMetadata) should be("Succeeded")
+            withClue(Option(getWorkflowFieldFromMetadata(notRunningMetadata, "failures")).getOrElse("No failures in metadata")) {
+              parseWorkflowStatusFromMetadata(notRunningMetadata) should be("Succeeded")
```

Comment:
Didn't you just rename this function?

Reply:
No, I added getWorkflowFieldFromMetadata